go.mod

```diff
@@ -158,6 +158,7 @@ require (
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.35.0 // indirect
 	go.opentelemetry.io/otel/sdk/metric v1.35.0 // indirect
 	go.opentelemetry.io/proto/otlp v1.5.0 // indirect
+	go.yaml.in/yaml/v3 v3.0.4 // indirect
```

Comment:
Ah looks like you are using this fork now but we are still importing `gopkg.in/yaml.v3` in https://github.com/docker/buildx/blob/7d0efdc50e218f3ffa97ba0d77a4c0d4fcf6dd57/bake/compose.go#L20

Can you change it there?

Also cc @thaJeztah @tonistiigi I see we have an indirect dep on https://github.com/moby/buildkit/blob/0396f3eda2c756696cfc773dc2cc15845891176b/go.mod#L185 so we have two sets for yaml module now. Didn't look yet which package imports it.

Reply:
Done!